src/main/java/hello/Application.java

```diff
@@ -10,4 +11,9 @@ public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
 	}
 
+	String generateSecretToken(){
+		Random r = new Random();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The use of java.util.Random is predictable [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=findsecbugs%3APREDICTABLE_RANDOM)